db/migrate/20220814230425_create_user_preferences.rb

```diff
@@ -0,0 +1,15 @@
+class CreateUserPreferences < ActiveRecord::Migration[7.0]
+  def change
+    create_table :user_preferences, if_not_exists: true do |t|
+      t.belongs_to :user, foreign_key: true, index: { unique: true }
+      t.boolean :auto_update_exercises, default: true, null: false
+
+      t.timestamps
+    end
+
+    # Run on Bastion in production
+    unless Rails.env.production?
+      User.find_each(&:create_preferences)
+    end
```

Comment:
We should run the create table and this block *before* deploying, and then the block again (with guards for `users.id>xxx`) once we're deployed. There will be a small window where we could break things for new users while that runs, so maybe we run it in a loop on bastion until we're fully deployed.